src/pyscript/_generator.py

```diff
@@ -50,6 +51,10 @@ def create_project(
     app_dir.mkdir()
     manifest_file = app_dir / config["project_config_filename"]
     with manifest_file.open("w", encoding="utf-8") as fp:
-        json.dump(context, fp)
+        if str(manifest_file).endswith(".json"):
```

Comment:
Naming nit? The constant is called "project_config_filename" but then we use "manifest_file" for the path? Maybe "project_config_path = ..."?